src/main/java/uk/gov/companieshouse/accounts/association/controller/UserCompanyAssociations.java

```diff
@@ -54,7 +54,7 @@ public ResponseEntity<AssociationsList> fetchAssociationsBy(
         LOG.infoContext(xRequestId, "Trying to fetch associations data for user in session :".concat(ericIdentity), null);
 
         final var user = usersService.fetchUserDetails(ericIdentity);
-        Optional.ofNullable(user).orElseThrow(() -> new BadRequestRuntimeException("Eric id is not valid"));//NOSONAR or else throw will be caught by controller advice
```

Comment:
hi may i know this line was removed. it was causing sonar issue, the line comment actually supresses sonar check.

Reply:
I think the comment was removed by accident. We can add it back.

I don't think the user can be null though, because the fetchUserDetails method will either return the user object, or result in some kind of exception like 404 exception if not found. Since the line is unreachable, I think this line can actually be removed completely.

Would you like us to add the comment back, or delete the line?

Reply:
Hi Kartheek, it was added back yesterday.